### ISSUE TITLE
PS 7.1 not showing in Windows Terminal - Showing PS 7.0.3 Instead

### ISSUE THREAD

**Issue by bahreex** [PowerShell/PowerShell#14535]
I am getting the Issue prompt shown in the attached image in the latest Windows Terminal when opening PS.

I have PS Core 7.1 already Installed. No Installation of PS 7.0.3 as I had already upgraded much earlier through Choco.

Not sure where this Issue is coming from. Pls help.

![image](https://user-images.githubusercontent.com/5578341/103489470-be238280-4e3a-11eb-98d2-27947daf4a8f.png)

## Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by DHowett** [comment]
Where is 7.1 installed? Where is 7.0.3 installed? You've clearly installed them side by side using different installation methods.

**Comment by bahreex** [comment]
> Where is 7.1 installed? Where is 7.0.3 installed? You've clearly installed them side by side using different installation methods.

Thanks for your response. 
Where do I find out about having them Installed side by side? In the system, I can only see 7.1 Installed within Add/Remo Programs and in Choco. can't find 7.0.3 anywhere.

![image](https://user-images.githubusercontent.com/5578341/103506934-fa2f0580-4e83-11eb-8f66-fff09c3e1b69.png)

![image](https://user-images.githubusercontent.com/5578341/103507045-36fafc80-4e84-11eb-84cb-f7ef4fea15d0.png)

**Comment by doctordns** [comment]
I have seen this challenge.  I just use the everywhere tool to search for pwsh.exe. 

See https://www.voidtools.com/

**Comment by bahreex** [comment]
> I have seen this challenge. I just use the everywhere tool to search for pwsh.exe.
> 
> See https://www.voidtools.com/
> 
> > <img alt="" width="48" src="/favicon.ico?v=2">**[voidtools](https://www.voidtools.com/)**

Thanks. I did use Everything and I could not find any 7.0.3 version.

![image](https://user-images.githubusercontent.com/5578341/103536710-097c7600-4eb9-11eb-91eb-d07a2c28c0c8.png)

**Comment by bahreex** [comment]
**bump**
Any help?

**Comment by iSazonov** [comment]
You could use Task Manager to find a path to pwsh being started from Terminal.

**Comment by jhoneill** [comment]
get-process -id $pid | % path ?
 
Will tell you where the current powershell has come from

And open Windows Terminal Settings and have a look at the values for commandline.

**Comment by bahreex** [comment]
Thanks, everyone. I resolved the issue. Problem was that installing 7.1 either from MSI or through Store was not overwriting the 7.0.3, and was also not Installing 7.1 side by side in any way. The 7.0.3 was also not visible in Add/Remove programs or in Choco or in Store. Since it was not visible (7.0.3), could not even uninstall it through any Uninstallers.

Some unknown Issue caused this somehow. 

I had to go into the C:\Program Files and manually delete the PowerShell folder. Then I had to Install 7.1 from MS Store, which Installed fine this time. The preview version is also through MS Store only, which shows up fine. Now there is no 7.0.3 in my system.

Note: Not sure Choco has both pwsh and powershell-core packages. They are the same but cause confusion.
